server/api.go

```diff
@@ -547,6 +599,10 @@ func (a *API) oauthRedirectHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	a.p.API.PublishWebSocketEvent(WSEventUserConnected, map[string]any{}, &model.WebsocketBroadcast{
```

Comment:
except for this line, all the changes in this method are related to the change allowing the "connect" button to work 😅